Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:3.6
+
+RUN apk --update add openjdk7-jre tar
```

Comment:
We are using jdk 8 to compile https://github.com/zilverline/apache-fop-server/blob/master/pom.xml#L235 so can we bump this to java 8?

Reply:
Ok. Updated and new image pushed (`zilverline/fop:1.7`)